v3/docs/gs/sysreqs.md

```diff
@@ -8,18 +8,25 @@ The following are system requirements for running HSE applications.
 Hardware requirements are largely dictated by the application embedding HSE
 and the amount of data stored.  The following are general guidelines.
 
-* **Architecture**: 64-bit Intel&reg; and AMD (x86_64); 64-bit IBM Z&reg; (s390x)
+* **Architecture**: 64-bit Intel&reg; or AMD (x86_64); 64-bit IBM Z&reg; (s390x)
 * **Memory**: 32 GB or more
 * **Block Storage**: SSD volumes *only*; use NVMe for best performance
 * **Persistent Memory** (optional): must support a DAX-enabled file system
 
-If a KVDB [media class](storage.md#media-classes) is configured on multiple
-block storage devices, such
-as when using XFS with LVM, performance can be **significantly** improved by
-balancing these devices across NUMA nodes.
-Tools like [`lstopo`](https://linux.die.net/man/1/lstopo) can
-be helpful in creating and verifying a balanced configuration.
+If a file system storing a KVDB [media class](storage.md#media-classes) is configured on a
+logical block device comprising multiple physical block devices, such as when using XFS with LVM,
+performance can be **significantly** improved by balancing these physical devices across NUMA nodes.
+Tools like [`lstopo`](https://linux.die.net/man/1/lstopo) can be helpful in creating and
```

Comment:
Might use the man page notation for `lstopo(1)`

Reply:
Done